app/views/procedures/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @procedures.each do |p|%>
```

Comment:
don't use one letter variables. use a descriptive variable name in this case `procedure`.